web/templates/layout/app.html.eex

```diff
@@ -1,3 +1,8 @@
+<%
+  title = "POST Bot for Slack"
```

Comment:
The strings should be pulled out and handled with gettext. The Logo url should be in a view (similar to a rails view helper). Just like with rails, we shouldn't be defining variables in the template.